native/cocos/editor-support/spine-wasm/spine-type-export.cpp

```diff
@@ -583,7 +612,29 @@ EMSCRIPTEN_BINDINGS(spine) {
         //.function("localToWorld", &Bone::localToWorld)
         .function("worldToLocalRotation", &Bone::worldToLocalRotation)
         .function("localToWorldRotation", &Bone::localToWorldRotation)
-        .function("rotateWorld", &Bone::rotateWorld);
+        .function("rotateWorld", &Bone::rotateWorld)
+        .function("setX", &Bone::setX)
```

Comment:
Need to verify these interfaces.

Reply:
Bone data is validated at the time of addition.
![Uploading Screenshot 2023-07-05 at 14.14.17.png…]()